app/models/change_password_form.rb

```diff
@@ -0,0 +1,18 @@
+class ChangePasswordForm
+  include ActiveModel::Model
+
+  attr_accessor :old_password, :password, :password_confirmation
+
+  validates_presence_of :old_password, :password, :password_confirmation
+  validates_confirmation_of :password
+
+  def initialize(params)
+    @old_password = params[:old_password]
+    @password = params[:password]
+    @password_confirmation = params[:password_confirmation]
+  end
+
+  def persisted?
```

Comment:
is this needed?

Reply:
My understanding is yes because you'd never want to persist the password form.  This ensure that if someone did try to use the persist method that it would always return false.

Reply:
it doesn't have a corresponding table to be persisted into, and additionally `false` is default for `persisted?` already

Reply:
https://apidock.com/rails/ActiveModel/Model/persisted%3F

Reply:
Removed